test/mpi/init/session_psets.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+#include "mpi.h"
+#include <stdio.h>
+#include "mpitest.h"
+
+/* This is adapted example 10.9 from MPI Standard 4.0 */
+
```

Comment:
Can we also copy the brief description of the example from standard and add into the header comment?